DNN Platform/Modules/TelerikRemoval/TelerikRemoval.dnn

```diff
@@ -0,0 +1,74 @@
+            </dependencies>
+            <components>
+                <component type="ResourceFile">
+                    <resourceFiles>
+                        <basePath>DesktopModules/TelerikRemoval</basePath>
+                        <resourceFile>
+                            <name>Resources.zip</name>
+                        </resourceFile>
+                    </resourceFiles>
+                </component>
+                <component type="Module">
+                    <desktopModule>
+                        <moduleName>TelerikRemoval</moduleName>
+                        <foldername>TelerikRemoval</foldername>
+                        <supportedFeatures/>
+                        <moduleDefinitions>
+                            <moduleDefinition>
+                                <friendlyName>Telerik Removal</friendlyName>
+                                <defaultCacheTime>0</defaultCacheTime>
+                                <moduleControls>
+                                    <moduleControl>
+                                        <controlKey/>
+                                        <controlSrc>DesktopModules/TelerikRemoval/View.ascx</controlSrc>
+                                        <supportsPartialRendering>False</supportsPartialRendering>
+                                        <controlTitle/>
+                                        <controlType>View</controlType>
+                                        <iconFile/>
+                                        <helpUrl>https://docs.dnncommunity.org/content/getting-started/setup/telerik-removal/index.html</helpUrl>
+                                        <viewOrder>0</viewOrder>
+                                    </moduleControl>
+                                </moduleControls>
+                            </moduleDefinition>
+                        </moduleDefinitions>
+                    </desktopModule>
+                </component>
+                <component type="Assembly">
+                    <assemblies>
+                        <assembly>
+                            <name>Dnn.Modules.TelerikRemoval.dll</name>
+                            <path>bin</path>
+                        </assembly>
+                        <assembly>
+                            <name>Dnn.Modules.TelerikRemovalLibrary.dll</name>
+                            <path>bin</path>
+                        </assembly>
+                    </assemblies>
+                </component>
+            </components>
+            <eventMessage>
+                <processorType>DotNetNuke.Entities.Modules.EventMessageProcessor, DotNetNuke</processorType>
+                <processorCommand>UpgradeModule</processorCommand>
+                <attributes>
+                    <businessControllerClass>Dnn.Modules.TelerikRemoval.UpgradeController, Dnn.Modules.TelerikRemoval</businessControllerClass>
+                    <upgradeVersionsList>upgrade</upgradeVersionsList>
```

Comment:
We only want to run the upgrade behavior on install. This should allow us to remove the change to `09.11.00.SqlDataProvider`.

```suggestion
                    <upgradeVersionsList>install</upgradeVersionsList>
```